fhir-model/src/main/java/com/ibm/fhir/model/path/FHIRPathType.java

```diff
@@ -298,6 +298,7 @@
     FHIR_VALUE_SET("FHIR", "ValueSet", FHIR_DOMAIN_RESOURCE, ValueSet.class),
     FHIR_VERIFICATION_RESULT("FHIR", "VerificationResult", FHIR_DOMAIN_RESOURCE, VerificationResult.class),
     FHIR_VISION_PRESCRIPTION("FHIR", "VisionPrescription", FHIR_DOMAIN_RESOURCE, VisionPrescription.class),
+    FHIR_UNKNOWN_RESOURCE_TYPE("FHIR", "UnknownResourceType"),
```

Comment:
maybe some documentation here so people know what this is and that its "special"?

Reply:
Added comments.